src/main/java/com/scylladb/cdc/debezium/connector/ScyllaExtractSimpleNewRecordState.java

```diff
@@ -0,0 +1,83 @@
+package com.scylladb.cdc.debezium.connector;
```

Comment:
Maybe this SMT should be moved to `com.scylladb.cdc.debezium.connector.transforms` - that way the package naming is consistent with other Debezium connectors.